docs/user-guide/src/capm3/pivoting.md

```diff
@@ -0,0 +1,166 @@
+
+This can now be achieved with the following procedure:
+
+1. Create a temporary bootstrap cluster, e.g. using Kind or Minikube using
+ `clusterctl`.
+
+2. Install ironic  components namely: ironic, ironic-inspector, ironic-database,
+   ironic-endpoint-keepalived, httpd and dnsmasq.
+
+3. Use clusterctl init to install the provider components
+
+   Example:
+
+   `clusterctl init --infrastructure metal3:v1.1.1
+   --target-namespace metal3 --watching-namespace metal3`
+
+ This command will create the necessary CAPI controllers (CAPI, CABPK, CAKCP)
+ and CAPM3 as the infrastructure provider. All of the controllers will be installed
+ on namespace `metal3` and they will be watching over objects in namespace `metal3`.
+
+4. Provision target cluster:
+
+   Example:
+
+   `clusterctl config cluster ... | kubectl apply -f -`
+
+5. Wait for the target management cluster to be up and running and once it is up
+ get the kubeconfig for the new target management cluster.
+
+6. Use the new cluster’s kubeconfig to install the ironic-components in the
+ target cluster.
+
+7. Use `clusterctl` init with the new cluster’s kubeconfig to install the provider
+ components.
+
+    Example:
+
+    `clusterctl init --kubeconfig target.yaml --infrastructure metal3:v0.3.1
+  --target-namespace metal3 --watching-namespace metal3`
+
+8. Use `clusterctl` move to move the Cluster API resources from the bootstrap
+ cluster to the target management cluster.
+
+    Example:
+
+    `clusterctl move --to-kubeconfig target.yaml -n metal3 -v 10`
+
+9. Delete the bootstrap cluster
+
+## Important Notes
+
+The following requirements are essential for the move process to run
+successfully:
+
+1. The move process should be done when the BMHs are in a steady state. BMHs
+   should not be moved while any operation is on-going i.e. BMH is in provisioning
+   state. This will result in failure since the interaction between IPA and ironic
+   gets broken and as a result ironic's database might not be repopulated and
+   eventually the cluster will end up in an erroneous state. Moreover, the IP of
+   the BMH might change after the move and the DHCP-leases from the management
+   cluster are not moved to target cluster.
+
+2. Before the move process is initialized, it is important to delete the ironic
+   pod/ironic containers. If ironic is deployed in cluster the deployment is named
+   `metal3-ironic`, if it is deployed locally outside the cluster, the containers
+   to be deleted are `ironic`, `ironic-inspector`, `ironic-endpoint-keepalived`,
+   `ironic-dnsmasq`, `ironic-httpd` and `mariadb`. If ironic is not deleted
+   before move, the old ironic might interfere with the operations of
+   the new ironic deployed in target cluster since the database of the first ironic
+   instance is not cleaned when the BMHs are moved. Also there would be two dnsmasq
+   existent in the deployment if there would be two ironic deployment which is
+   undesirable.
```

Comment:
```suggestion
2. Before the move process is initialized, it is important to delete the Ironic
   pod/ironic containers. If Ironic is deployed in cluster the deployment is named
   `metal3-ironic`, if it is deployed locally outside the cluster, the containers
   to be deleted are `ironic`, `ironic-inspector`, `ironic-endpoint-keepalived`,
   `ironic-dnsmasq`, `ironic-httpd` and `mariadb`. If Ironic is not deleted
   before move, the old ironic might interfere with the operations of
   the new ironic deployed in target cluster since the database of the first Ironic
   instance is not cleaned when the BMHs are moved. Also there would be two dnsmasq
   existent in the deployment if there would be two ironic deployment which is
   undesirable.
```

Reply:
done